.github/workflows/tests.yml

```diff
@@ -304,3 +304,113 @@ jobs:
           curl https://github.com/googleapis/repo-automation-bots/releases/download/flakybot-1.1.0/flakybot-darwin-amd64 -o flakybot -s -L
           chmod +x ./flakybot
           ./flakybot --repo ${{github.repository}} --commit_hash ${{github.sha}} --build_url https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+
+  sample-tests:
+    # run job on proper workflow event triggers (skip job for pull_request event from forks and only run pull_request_target for "tests: run" label)
+    if: "${{ (github.event.action != 'labeled' && github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) || github.event.label.name == 'tests: run' }}"
+    name: Sample tests
+    strategy:
+      matrix:
+        node-version: [v18.x]
+        os: [ubuntu-latest]
+      fail-fast: false
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+      issues: write
+      pull-requests: write
```

Comment:
are all of these permissions reqquired? `issues` and `pull_request` are for flakybot... we are not running flakybot here are we?

Reply:
worth maybe testing removing some...

Reply:
no need to run flakybot on these, I'll remove `issues` and `pull_request` then!